Dockerfile

```diff
@@ -1,3 +1,16 @@
+# Build ostreeuploader, aka fiopush/fiocheck
+FROM ubuntu:20.04
+RUN apt-get update
+RUN apt-get install -y wget git gcc make -y
+RUN wget -P /tmp https://go.dev/dl/go1.18.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf /tmp/go1.18.linux-amd64.tar.gz
+ENV PATH /usr/local/go/bin:$PATH
+
+RUN git clone https://github.com/foundriesio/ostreeuploader.git /ostreeuploader && \
+    cd /ostreeuploader && git checkout ae83a2188808fd6188adb25ac72dffd4beaadd3d && \
```

Comment:
Can we have a tag to use instead of a fixed hash?

Reply:
updated.